pkg/dependenciesdistributor/dependencies_distributor.go

```diff
@@ -151,7 +151,6 @@ func (d *DependenciesDistributor) OnUpdate(oldObj, newObj interface{}) {
 		klog.V(4).Infof("Ignore update event of object (%s, kind=%s, %s) as specification no change", unstructuredOldObj.GetAPIVersion(), unstructuredOldObj.GetKind(), names.NamespacedKey(unstructuredOldObj.GetNamespace(), unstructuredOldObj.GetName()))
 		return
 	}
-	d.OnAdd(oldObj)
```

Comment:
Currently, we support the use of labelselector to follow resource selection. When a resource is updated, the label of the resource may change. Therefore, the resource may meet the requirements before the update but not after the update. If we delete this line, we might miss this sceneary.

Reply:
So under the cleanup of labelsSelector unchanged, it can be deleted

Reply:
Yes, it is.